distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -14,14 +14,8 @@
 -->
 <Configuration status="FATAL">
     <Properties>
-        <Property name="color.fatal">bright red</Property>
-        <Property name="color.error">bright red</Property>
-        <Property name="color.warn">bright yellow</Property>
-        <Property name="color.info">bright green</Property>
-        <Property name="color.debug">cyan</Property>
-        <Property name="color.trace">cyan</Property>
-        <Property name="log-pattern">%-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n</Property>
-        <Property name="log-out-pattern">\u001b[90m%d{HH:mm:ss\.SSS}\u001b[0m %highlight{%-5level}{FATAL=${color.fatal}, ERROR=${color.error}, WARN=${color.warn}, INFO=${color.info}, DEBUG=${color.debug}, TRACE=${color.trace}} \u001b[90m[%t]\u001b[0m %msg%n%throwable</Property>
```

Comment:
The color properties and new log pattern were added when Karaf was upgraded to 4.2.0 (#3828) but they are not being used.